<?xml version="1.0"?>
<launch> 


    <node name="loam_feature_extractor_node" pkg="spinning_lidar_loam" type="loam_feature_extractor" output="screen" >
        <param name="min_dist_to_sensor" value="5.0" />
        <param name="laser_link" type="string" value="laser" />
        <param name="base_link" type="string" value="laser_mount" />
        <param name="laser_scan_topic" type="string" value="spinning_lidar/scan" />
        <param name="filtered_scan_topic" type="string" value="spinning_lidar/filtered_scan" />
        <param name="ir_interrupt_topic" type="string" value="spinning_lidar/ir_interrupt" />
    </node>

	

</launch>